scripts/actions/weaponskills/final_paradise.lua

```diff
@@ -13,7 +13,7 @@ local weaponskillObject = {}
 
 weaponskillObject.onUseWeaponSkill = function(player, target, wsID, tp, primary, action, taChar)
     local params = {}
-    params.numHits = 2
+    params.numHits = 1
```

Comment:
https://www.bg-wiki.com/ffxi/Final_Paradise
The description of this WS says its two hits though I can't verify...